docs/misc/12_support_and_sla/index.mdx

```diff
@@ -0,0 +1,63 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Support & SLA
+
+Windmill is committed to delivering comprehensive support and maintaining superior service levels. Our Support & Service Level Agreements (SLAs) are specifically designed to provide timely assistance and address any issues or concerns that may arise during your usage of the Windmill platform.
+
+## Technical Support
+
+Our knowledgeable support team is available 24/7 to assist you with any technical inquiries, troubleshooting, or questions you may have. We are committed to providing timely responses and resolutions to ensure a seamless experience with Windmill.
+
+## Issue Prioritization
+
+To effectively handle and prioritize issues, we categorize them into different priority levels based on their impact and urgency. This enables us to allocate resources efficiently and ensure that critical issues receive immediate attention.
+
+- P0 (Priority 0): Reserved for blocking and urgent issues that require immediate resolution.
+- P1 (Priority 1): High-priority issues that significantly impact functionality or performance.
+- P2 (Priority 2): Medium-priority issues that have a moderate impact on functionality.
+- P3 (Priority 3): Low-priority issues that have minimal impact on functionality or usability.
+
+## Service Level Agreement (SLA)
+
+Our SLA defines the expected response times for each priority level:
+
+- P0: We strive to respond within the same business day.
+- P1: Our goal is to respond within 24 hours.
+- P2: We target a response time of 48 hours.
+- P3: We aim to respond within 72 hours.
```

Comment:
4 ways of saying the same things, extract the text above and just put time instead

you can put 3h in p0